tests/test_config.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from os import path
+
+import yaml
+from kedro.config.config import MissingConfigException
+
+from kedro_kubeflow.config import PluginConfig
+
+
+class TestPluginConfig(unittest.TestCase):
+    def test_plugin_config(self):
+
+        test_dir = path.dirname(path.abspath(__file__))
+        with open(path.join(test_dir, "test_config.yml"), "r") as stream:
```

Comment:
Consider providing config file as string here, so the "given" context is closer to the assumptions (it's easier to extend the test this way)